assig1/assignment-1/src/main/java/ro/utcn/spet/example/a1/controller/ConsoleController.java

```diff
@@ -0,0 +1,435 @@
+package ro.utcn.spet.example.a1.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import ro.utcn.spet.example.a1.entity.*;
+import ro.utcn.spet.example.a1.exception.QuestionNotFoundException;
+import ro.utcn.spet.example.a1.exception.StackUserNotFoundException;
+import ro.utcn.spet.example.a1.service.*;
+
+import java.security.PrivilegedAction;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static ro.utcn.spet.example.a1.entity.VoteValue.DOWN;
+import static ro.utcn.spet.example.a1.entity.VoteValue.UP;
+import static ro.utcn.spet.example.a1.entity.VoteValue.ZERO;
+
+@Component
+@RequiredArgsConstructor
+// Command line runners are executed by Spring after the initialization of the app has been done
+// https://www.baeldung.com/spring-boot-console-app
+public class ConsoleController implements CommandLineRunner {
```

Comment:
Overall, you have a lot of business logic inside this class (like vote calculation, some business rules, etc). You should move this to a service class.